arb_os/codeSegment.mini

```diff
@@ -276,7 +272,7 @@ public impure func translateEvmCodeSegment(
                         if (opcode == 0x43) { // NUMBER
                             return pushEvmInsnCall(evmOp_number, restOfCode, evmJumpTable);
                         } elseif (opcode == 0x44) { // DIFFICULTY
-                            return Some((pushValue(0, restOfCode), evmJumpTable));
+                             return Some((restOfCode, evmJumpTable));
```

Comment:
What does this do?

Reply:
It's a no-op.

Reply:
The difficulty opcode is expected to push an integer to the stack. We either need to reject code that includes the difficulty opcode, or have the opcode push some integer

Reply:
Hmm, somehow I misread the spec.  Fixing this to push an integer reflecting a recent difficulty level.